llvm/lib/CodeGen/MachineOutliner.cpp

```diff
@@ -593,6 +593,9 @@ void MachineOutliner::findCandidates(
     unsigned NumDiscarded = 0;
     unsigned NumKept = 0;
 #endif
+    // Sort the start indices so that we can efficiently check if candidates
+    // overlap with each other in MachineOutliner::findCandidates().
```

Comment:
I'm a bit confused why mention this function's name here (`MachineOutliner::findCandidates()`). From the wording it would appear that it would referring to another function. Perhaps "overlap with each other further down" will be clearer.

Reply:
Thanks for the comments. Modified!